src/js/Home.js

```diff
@@ -9,18 +9,24 @@ import React from "react";
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
-
 import Country from './Country';
+import HomePageSelector from './HomePageSelctor';
+
 
 const Home = props => {
+
+    const countryName = localStorage.getItem('countryName');
```

Comment:
Ideally this country name should be a state variable.. it can take value from either localstorage or the popup slected value..

The state value get passed to the Country component as    props.. Thats all